catalogs/catalog-jdbc-mysql/src/test/java/com/datastrato/gravitino/catalog/mysql/integration/test/CatalogMysqlIT.java

```diff
@@ -1264,7 +1264,8 @@ void testNameSpec() {
     Assertions.assertTrue(
         Arrays.stream(schemaIdents).anyMatch(s -> s.name().equals(testSchemaName)));
 
-    Assertions.assertTrue(catalog.asSchemas().dropSchema(schemaIdent, false));
+    Assertions.assertTrue(
+        catalog.asSchemas().dropSchema(schemaIdent, false), "schema should be dropped");
```

Comment:
I notice that `schema should be dropped` use many times. Should we define a method in ITUtils?